### DISCUSSION TITLE
Change run time from readable to epoch

### DISCUSSION THREAD

**Discussion by s0c-org** [masonr/yet-another-bench-script#46] [Ideas]
I would like to start the discussion that we change the readable time in the JSON output to epoch for API requests. 

Advantages that would come from this: 

- Easier processing in programming languages. 
- More reliable when converting to other time zones.

If you would go a bit further with this idea and add the epoch time from start and end, it would be possible with very little effort to implement a stopwatch for the benchmark.

**Comment by masonr** [comment]
Agreed with your idea!

PR #51 was just merged, which records the epoch start/stop times of the script and calculates the runtime. I'll be adding these values and the duration to the JSON output soon.
